pkgs/development/python-modules/mung/default.nix

```
@@ -0,0 +1,43 @@
+{ lib
+, buildPythonPackage
+, fetchFromGitHub
+, lxml
+, matplotlib
+, midiutil
+, numpy
+, pytestCheckHook
+, scikitimage
+, scikit-learn
+}:
+
+buildPythonPackage rec {
```

```suggestion
buildPythonPackage {
```

No recursion required here.